tidb/run

```diff
@@ -0,0 +1,6 @@
+lein run test \
```

Comment:
1. If we want to let SRE team run multi cases automatically, how do they do? 
2. If I want to run some other cases, how can I know the real name? Review the codes and find the correct one? 

The script is too simple.